packages/core/src/components/Review/Review.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReviewLink from './ReviewLink';
+import classNames from 'classnames';
+
+/*
+`<Review>`
+
+@react-component Review
+
+Style guide: components.review.react
+*/
+
+/**
+ * The `Review` component is used for listing a summary of information entered
+ * by a user. Its content includes a heading, value, and edit link.
+ */
+export class Review extends React.PureComponent {
+  heading() {
+    if (this.props.heading) {
+      return (
+        <h3 className="ds-c-review__heading ds-text ds-u-margin-bottom--0 ds-u-font-weight--bold ds-u-display--inline-block">
+          {this.props.heading}
+        </h3>
+      );
+    }
+  }
+
+  render() {
+    const classes = classNames(
+      'ds-c-review ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex',
+      !this.props.alignTop && 'ds-u-align-items--center'
+    );
+    return (
+      <div className={classes}>
+        <div className="ds-u-margin-right--2">
+          {this.heading()}
+          <div className="ds-c-review__body">{this.props.children}</div>
+        </div>
+        <ReviewLink onClick={this.props.onClick} href={this.props.href}>
+          {this.props.linkText}
+        </ReviewLink>
+      </div>
+    );
+  }
+}
+
+Review.defaultProps = {
+  alignTop: true,
+  linkText: 'Edit'
+};
+
+Review.propTypes = {
+  /**
+   * Set to false to vertically center the edit link.
+   */
+  alignTop: PropTypes.bool,
```

Comment:
Do y'all use this prop anywhere currently? I don't think we have a need for it on App 3, and from a design standpoint, I'd almost prefer not offering this option so directly. If we add support for a `className` prop, someone could pass in `ds-u-align-items--center` if they really wanted it.

For reference though, in case we do want to keep this: in other components a boolean props presence means `true`. (ie. `alignCenter` here instead). This is so its consistent with other HTML attributes like `selected`, `required`, and `checked`.

Reply:
There was a vets.gov screenshot on the original ticket with the edit link vertically centered, but in tax tools it is aligned to the top. I'm happy to remove that prop and just have the user add that via their own class.

Reply:
Ah, that's my fault, sorry for the confusion. Yeah, let's get rid of this for now. We're not vertically centering this anywhere right now. I'd still advocate for adding support for a `className` prop though 😄